### DISCUSSION TITLE
Completed course work, IM-11

### DISCUSSION THREAD

**Discussion by LLumisS** [HowProgrammingWorks/LiveQA#94] [Code review]
Course work is done ✅

Kirill Bukach, IM-11.
Procedural Map Generation - JavaScript.
Main branch: 'master'.

Link: https://github.com/LLumisS/Procedural-Map-Generation-Project

**Comment by tshemsedinov** [comment]
See review: https://github.com/LLumisS/CourseWork/issues/1
